JKBTumpalan/evtf_project
# Feature/pages mobilenet classifier

# Ticket
- None

# Changes
- MobileNet classifier API endpoint
- Docker compose updates
- Animal Classifier (MobileNet) page behavior and state management

# Dependencies
None

# Notes
- None

# Test Plan
1. Pull branch
2. change working directory to **_/evtf_project_**
3. `docker-compose up --build`
- [ ] Check if the animal classifier is working both via upload and via image link input
- [ ] Check if the input area updates whenever image link or upload button is selected
- [ ] Check if the image preview is present for both image input 
- [ ] Check if the labels show after prediction
- [ ] Check if the animal classifier is accurate in detecting the animals
